pages/_language/events/_event/index.vue

```diff
@@ -8,7 +8,13 @@
         :all-locales="internalEvent._allNameLocales"
       />
 
-      <event-banner v-bind="internalEvent" />
+      <event-banner
+        :name="internalEvent.name"
+        :location="internalEvent.location"
+        :visually-hide-name="internalEvent.visuallyHideName"
+        :display-date="internalEvent.displayDate"
+        :banner-icon="internalEvent.bannerIcon"
+      />
```

Comment:
Why spread the attributes here?

Reply:
Because that is also done on the `event-header` component.

Reply:
But the `event-header` component is not functional, `v-bind` is also used at the bottom of this page for the footer.

Reply:
Think it is nice to use `v-bind` for 'dumb components' since then changing data props don't have to happen at two places.

Reply:
Because that is also done on the event-header component.

Reply:
Ah, agreed.